### PR TITLE
Changes for the Team page

### PR DESCRIPTION
**Pull request by sbenito**
Tried to use plain Markdown but ended up embedding HTML to add text with links to the pictures. 

The initial list of people, title and functions:

Sara Benito M.Sc., Biomedical Software Engineer
Weiwei Chu M.Sc., Database Manager
Dongjin Kwon Ph.D., Research Scientist [http://web.stanford.edu/~djkwon/]
Nolan Nichols Ph.D., Postdoctoral Fellow [http://www.nolan-nichols.com/]
San Hyun Park Ph.D., Postdoctoral Fellow [https://web.stanford.edu/~shpark1/]
Adolf Pfefferbaum  M.D., Center Director [https://www.sri.com/about/people/adolf-pfefferbaum]
Kilian Pohl Ph.D., Program Director [http://web.stanford.edu/~kpohl/]
Edith Vioni Sullivam M.D., Ph.D. Professor [https://med.stanford.edu/profiles/edith-sullivan/]

### CODE REVIEW

File: content/pages/team.md [hunk 1]
```diff
@@ -1,4 +1,75 @@
...
+                </a>
+                <b>Program Director</b>
+</div>
+
+---
+
+<div>
+        <img src='../images/team/edith.png' hspace='20'>
+                <a href='https://med.stanford.edu/profiles/edith-sullivan/'>
+                        <strong>Edith Vioni Sullivam M.D.</strong>
```
Comment (nicholsn) [inline content/pages/team.md]: only PhD no MD

**Comment by nicholsn** [comment]
@sbenito any reason not to use markdown? It would be nice use to keep things consistent unless there is some formatting that cannot be completed with it

https://daringfireball.net/projects/markdown/syntax

File: content/pages/team.md [hunk 2]
```diff
@@ -1,4 +1,75 @@
...
 Author: Nolan Nichols <http://orcid.org/0000-0003-1099-3328>
 Summary: Team members of the SIBIS research group
+#Team
+---
+<div>
+	<img src='../images/team/sara.png' hspace='20'>
+		<strong>Sara Benito Arce</strong>,
+		<b>Biomedical Software Engineer</b>
+</div>
+---
```
Comment (nicholsn) [inline content/pages/team.md:12]: For example, this:

```
![Alt Sara Benito](https://raw.githubusercontent.com/sibis-platform/sibis-platform.github.io/c339e022d4be01a03823da4816d5d366d22261ef/content/images/team/sara.png)
__Sara Benito Arce__
_Biomedical Software Engineer_
```

Produces this:

Alt Sara Benito
**Sara Benito Arce**
_Biomedical Software Engineer_

**Comment by sbenito** [comment]
@nicholsn 
Yes, you can add text to images, but no text with links.

**Comment by nicholsn** [comment]
not sure I follow, but should be ok.

Alt Sara Benito
**[Sara Benito Arce](http://web.stanford.edu/~djkwon/)**
_Biomedical Software Engineer_

File: content/pages/team.md [hunk 3]
```diff
@@ -1,4 +1,75 @@
...
+</div>
+
+---
+<div>
+        <img src='../images/team/djk.png' hspace='20'>
+                <a href='http://web.stanford.edu/~djkwon/'>
+			<strong>Dongjin Kwon Ph.D.,</strong>
+		</a>
+		<b> Research Scientist</b>
+</div>
```
Comment (nicholsn) [inline content/pages/team.md:25]: only other thing is closing the image tag `</img>`

Comment (sbenito) [reply content/pages/team.md:25]: http://www.w3schools.com/tags/tag_img.asp

Comment (nicholsn) [reply content/pages/team.md:25]: ok, guess just confused why the following links were indented

Comment (sbenito) [reply content/pages/team.md:25]: Right, I should be getting an editor.

**Comment by sbenito** [comment]
I can change and do that, but I liked the text on the side.

**Pull request merged**